buildSrc/subprojects/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/DetermineBaselines.kt

```diff
@@ -50,7 +51,7 @@ abstract class DetermineBaselines @Inject constructor(@get:Internal val distribu
             determinedBaselines.set(defaultBaseline)
         } else if (configuredBaselines.getOrElse("") == flakinessDetectionCommitBaseline) {
             determinedBaselines.set(determineFlakinessDetectionBaseline())
-        } else if (!currentBranchIsMasterOrRelease() && configuredBaselines.isDefaultValue()) {
+        } else if (!currentBranchIsMasterOrRelease() && !OperatingSystem.current().isWindows && configuredBaselines.isDefaultValue()) {
             determinedBaselines.set(forkPointCommitBaseline())
```

Comment:
Why not do this on Windows, and what's the consequence of this?

Reply:
I don't know, let's remove it to see if anything breaks. My assumption is that it ran into a long path issue.

Reply:
Hmmmmm...`git` can't checkout the source because of long path issues...

Reply:
Please explain the reasons for this in a comment.